packages/ui-view/src/View/__tests__/View.test.js

```diff
@@ -55,10 +53,10 @@ describe('<View />', async () => {
 
   it('should pass whitelisted style attributes', async () => {
     const styleProps = {
-      top: '10rem',
```

Comment:
Why is the change from rem to px? Just wondering

Reply:
Because `getComputedStyle` for an element will give back the computed CSS values (eg. 10 rem would be around 160px) I thought its easier to see what is happening in the test.